src/server/scripts/Events/hallows_end.cpp

```diff
@@ -0,0 +1,536 @@
+                                    DoCast(trigger, SPELL_START_FIRE);
+                            _events.Repeat(Seconds(5));
+                        }
+                        break;
+                    }
+                    case EVENT_STOP_APPLYING_FIRE:
+                        _canShootFire = false;
+                        break;
+                    case EVENT_FINISH:
+                    {
+                        bool failed = false;
+                        std::list<Creature*> triggerList;
+                        me->GetCreatureListWithEntryInGrid(triggerList, NPC_FIRE_TRIGGER);
+                        for (Creature* trigger : triggerList)
+                        {
+                            if (trigger->HasAura(SPELL_FIRE_AURA_BASE))
+                                failed = true;
+
+                            trigger->AI()->EnterEvadeMode();
+                        }
+                        FinishEvent(failed);
+                        break;
+                    }
+                    case EVENT_CLEAVE:
+                        DoCastVictim(SPELL_CLEAVE);
+                        _events.Repeat(Seconds(20));
+                        break;
+                }
+
+            }
+
+            if (!UpdateVictim())
+                return;
+
+            DoMeleeAttackIfReady();
+        }
+
+        void FinishEvent(bool failed)
+        {
+            _events.Reset();
+            if (failed)
+            {
+                Talk(SAY_FAIL);
+                me->DespawnOrUnsummon();
+            }
+            else
+            {
+                Talk(SAY_FLAMES_DOWN);
+                float x, y, z;
+                GetPosToLand(x, y, z);
+                me->GetMotionMaster()->MovePoint(POINT_HORSEMAN_ATTACK, x, y, z);
+            }
+        }
+
+        void MovementInform(uint32 type, uint32 point) override
+        {
+            if (type == POINT_MOTION_TYPE && point == POINT_HORSEMAN_ATTACK)
+            {
+                me->SetFlag(UNIT_FIELD_FLAGS, 0);
+                me->SetDisableGravity(false);
+                me->RemoveAurasDueToSpell(SPELL_HORSEMAN_MOUNT);
+                if (Unit* target = me->SelectNearestPlayer(30.0f))
+                    AttackStart(target);
+            }
+        }
+
+        void JustDied(Unit* /*killer*/) override
+        {
+            Talk(SAY_DEFEATED);
+            float x, y, z;
+            GetPosToLand(x, y, z);
+            me->CastSpell(x, y, z, SPELL_SUMMON_LANTERN, true);
+            CompleteQuest();
+        }
+
+        void CompleteQuest()
+        {
+            float radius = 100.0f;
+            std::list<Player*> players;
+            Trinity::AnyPlayerInObjectRangeCheck checker(me, radius);
+            Trinity::PlayerListSearcher<Trinity::AnyPlayerInObjectRangeCheck> searcher(me, players, checker);
+            me->VisitNearbyWorldObject(radius, searcher);
```

Comment:
```
src/server/scripts/Events/hallows_end.cpp:398:17:
 fatal error: no member named 'VisitNearbyWorldObject' in 'Creature'
            me->VisitNearbyWorldObject(radius, searcher);
            ~~  ^
1 error generated.
```
I would think this needs another `#include` , but I can't find `VisitNearbyWorldObject()` in the source at all.

Reply:
Meh, it has been changed to `VisitNearbyObject` about the time this PR was started. :-P